packages/components/inputs/selectable-search-input/README.md

```diff
@@ -0,0 +1,150 @@
+| `maxMenuHeight`            | `ReactSelectProps['maxMenuHeight']`                                                    |          |           | Maximum height of the menu before scrolling&#xA;<br>&#xA;[Props from React select was used](https://react-select.com/props)                                                                                                                                                                                                                                                              |
+| `closeMenuOnSelect`        | `ReactSelectProps['closeMenuOnSelect']`                                                |          |           | Whether the menu should close after a value is selected. Defaults to `true`.&#xA;<br>&#xA;[Props from React select was used](https://react-select.com/props)                                                                                                                                                                                                                             |
+| `menuHorizontalConstraint` | `union`<br/>Possible values:<br/>`3 , 4 , 5`                                           |          | `3`       | Horizontal size limit for the dropdown menu.                                                                                                                                                                                                                                                                                                                                             |
+| `showSubmitButton`         | `boolean`                                                                              |          | `true`    | Show submit button in the input                                                                                                                                                                                                                                                                                                                                                          |
+
+## Signatures
+
+### Signature `value`
+
+```ts
+{
+  text: string;
+  option: string;
+}
+```
+
+### Signature `onChange`
+
+```ts
+(event: TCustomEvent) => void
+```
+
+### Signature `onBlur`
+
+```ts
+(event: TCustomEvent) => void
+```
+
+### Signature `onFocus`
+
+```ts
+(event: TCustomEvent) => void
+```
+
+### Signature `onSubmit`
+
+```ts
+(value: TValue) => void
+```
+
+### Signature `onReset`
+
+```ts
+() => void
+```
+
+## `data-*` props
+
+The component further forwards all `data-` attributes to the underlying `input` component.
+
+## Static methods
+
+### `SelectableSearchInput.isEmpty`
+
+Returns `true` when the value is considered empty, which is when the value is empty or consists of spaces only.
+
+```js
+SelectableSearchInput.isEmpty(''); // -> true
```

Comment:
I think this needs to be updated taking [this](https://github.com/commercetools/ui-kit/pull/2426/files#diff-c0601f2ee89d0052b13493f19c29394110f25ef8689fa42ba3f7addb227e7ff6R73) API into consideration.

Reply:
Thanks, [e811206](https://github.com/commercetools/ui-kit/pull/2426/commits/e811206c4f7c216e22f85871c4a16990c6ea9188)